bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/internal/components/UIComponentSitemapProvider.java

```diff
@@ -79,6 +85,13 @@ public class UIComponentSitemapProvider implements SitemapProvider, RegistryChan
     private static final String SITEMAP_PREFIX = "uicomponents_";
     private static final String SITEMAP_SUFFIX = ".sitemap";
 
+    private static final String COMPARATOR_STRING = "==|!=|<=|>=|<|>";
+    private static final Pattern COMPARATOR = Pattern.compile(COMPARATOR_STRING);
```

Comment:
I think you can remove these two now, correct?

Reply:
Indeed